components/Banner/tests/unit/Theming/MDCBannerViewThemingTests.m

```diff
@@ -205,6 +208,31 @@ - (void)testFontsAreScaledWhenTypographySchemeRequestsPrescaling {
   }
 }
 
+- (void)testBannerViewBackgroundColorChangeWhenUIUserInterfaceStyleChangeOnIOS13 {
```

Comment:
Nit: Add an "s" to "Change" in this test name ( should be `testBannerViewBackgroundColorChangeWhenUIUserInterfaceStyleChangesOnIOS13`)

Reply:
Addressed in 94eb60abbf3abe80848d1c94775d274fad881f24.